terraform/digitalocean.sample.tf

```diff
@@ -13,6 +13,6 @@ module "do-hosts" {
   region_name = "nyc3" # this must be a region with metadata support
 
   control_count = 3
-  worker_count = 3
+  worker_count = 4
```

Comment:
Is the increase from 3 to 4 worker nodes necessary? I've been able to run ELK with 3 just fine on DO, several times.